NorthstarDLL/CMakeLists.txt

```diff
@@ -149,11 +150,12 @@ add_library(NorthstarDLL SHARED
             "dllmain.cpp"
             "dllmain.h"
             "ns_version.h"
-)
+ "mods/autodownload/moddownloader.h" "mods/autodownload/moddownloader.cpp")
```

Comment:
could this be put onto two different lines?

Reply:
Done in 15043b6d01d80c889d412cc46a4eef57ce4175ea.